server/src/main/java/org/apache/iotdb/db/service/DataNode.java

```diff
@@ -316,6 +315,7 @@ private void setUp() throws StartupException, QueryProcessException {
     registerManager.register(RegionMigrateService.getInstance());
 
     registerManager.register(MetricService.getInstance());
+    registerManager.register(CompactionTaskManager.getInstance());
```

Comment:
As we change the sequence of invoking `CompactionTaskManager.getInstance()`, why do we need to add the variable `init` inside the class ?